ui/app/AppLayouts/Chat/ChatColumn/SuggestionFilter.qml

```diff
@@ -69,41 +76,19 @@ Item {
                 (hasAtBeforeCursor && hasWhiteSpaceBeforeAt && hasWhiteSpaceAfterAt) ||
                 (this.cursorPosition === 1 && hasAtBeforeCursor && hasWhiteSpaceAfterAt) ||
                 (cursorAtEnd && filter.endsWith("@") && hasWhiteSpaceBeforeAt)) {
-            this.lastAtPosition = this.cursorPosition - 1;
             return true
         }
 
         return false
     }
 
-    function isFilterReady(filter) {
-        if (filter === undefined) {
-            return false
-        }
-
-        // Prevents suggestions to show up at all
-        if (filter.indexOf("@") === -1)  {
-          return false
-        }
-
-        let hasWhiteSpaceBeforeCursor = filter.charAt(this.cursorPosition - 1) === " "
-
-        if (filter.charAt(this.cursorPosition - 2) === "@" && hasWhiteSpaceBeforeCursor) {
-            return false
-        }
```

Comment:
I don't think I can see this condition in the new change but I am also not sure if this is a valid one!

Reply:
I found this condition very suspicious and error-prone, so I removed it. Then I checked if the feature is not broken and it looked good. So I decided to keep this removed.

Reply:
Yeah I had similar feeling when reading it but as you tested and looks good, it seems legit!